### ISSUE TITLE
Coinmarketcap.com

### ISSUE THREAD

**Issue by dammas1816** [mobnetic/BitcoinChecker#265]
Please add this crypto value viewer!!!

**Comment by pesho** [comment]
+1000, this is the most reliable price source for me. They monitor multiple markets/exchanges to calculate an average price, thus eliminating local exchange fluctuations.

**Comment by pesho** [comment]
They have an API as well: http://coinmarketcap.com/api/

**Comment by KJayy** [comment]
Yeah adding the coinbase API would actually be quite incredible. That way you could always get Fiat value of any coin.
